Exiled.Events/Patches/Events/Player/SpawningRagdoll.cs

```diff
@@ -43,12 +43,22 @@ private static bool Prefix(RagdollManager __instance, ref Vector3 pos, ref Quate
                 ownerNick = ev.PlayerNickname;
                 playerId = ev.PlayerId;
 
+                if (!ev.IsAllowed)
+                    return false;
+
                 Role role = __instance.hub.characterClassManager.Classes.SafeGet(classId);
+                if (role.model_ragdoll == null)
+                    return false;
                 GameObject gameObject = UnityEngine.Object.Instantiate(role.model_ragdoll, pos + role.ragdoll_offset.position, Quaternion.Euler(rot.eulerAngles + role.ragdoll_offset.rotation));
+                Ragdoll component = gameObject.GetComponent<Ragdoll>();
```

Comment:
You are still instantiating a new GameObject, and then running a GetComponent<> without that componend ever being added anywhere.

Have you tested this and made sure that this GetComponent<> isn't null?

Reply:
I've tested it and everything works. Tried to enable/disable/get ragdoll informations with commands in game.
I'll add a null-check today/tomorrow, but since I spawn the ragdoll using the API.Features.Ragdoll class, the GameObject can't be null, or it wouldn't be spawned.